backend/orchestrator/functions/requestSyncDeployment/handler.ts

```diff
@@ -1,12 +1,11 @@
 import { ulid } from 'ulid';
 
 import { requestSyncDeployment } from '@swarmion/orchestrator-contracts';
-import { getLambdaHandler } from '@swarmion/serverless-contracts';
-import { applyHttpMiddlewares } from '@swarmion/serverless-helpers';
+import { getHttpLambdaHandler } from '@swarmion/serverless-contracts';
 
 import ServiceEventEntity from 'libs/dynamodb/models/serviceEvent';
 
-const handler = getLambdaHandler(requestSyncDeployment)(async event => {
+export const main = getHttpLambdaHandler(requestSyncDeployment)(async event => {
```

Comment:
We lose the validation of the `inputSchema` that we add with `applyHttpMiddlewares` + `inputSchema` here though

Reply:
Gottem haha
It was done in the upcoming commits 😮‍💨